core/src/main/scala/tech/ant8e/uuid4cats/UUID4cats.scala

```diff
@@ -18,7 +18,7 @@ package tech.ant8e.uuid4cats
 
 import cats.effect.std.{Mutex, Random, SecureRandom}
 import cats.effect.{Async, Clock, Ref}
-import cats.implicits._
+import cats.implicits.*
```

Comment:
Since cats 2.2.0, the recommended import is `cats.syntax.all._` as it compiles faster.

Also, `*` breaks Scala 2?